src/components/modules/ModuleCheckout.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from "react";
+import { View } from "react-native";
+import Placeholder from "rn-placeholder";
+import {
+  IOColors,
+  IOModuleIDPSavedVSpacing,
+  IOModuleStyles,
+  useIOExperimentalDesign
+} from "../../core";
+import { IOLogoPaymentType, LogoPayment } from "../logos";
+import { VSpacer } from "../spacer";
+import { Body, H6 } from "../typography";
+import { PressableModuleBase } from "./PressableModuleBase";
+
+type ModuleCheckoutProps = { ctaText: string } & (
+  | {
+      paymentLogo?: IOLogoPaymentType;
+      header: string;
+      body: string;
+      isLoading?: false;
+      onPress?: () => void;
+    }
+  | {
+      isLoading: true;
+    }
+);
+
+type CtaOnlyProps = { text?: string };
+
+export const ModuleCheckout = (props: ModuleCheckoutProps) => {
+  if (props.isLoading) {
+    return <LoadingVersion text={props.ctaText} />;
+  }
+  return (
+    <PressableModuleBase onPress={props.onPress}>
+      {props.paymentLogo && <LogoPayment name={props.paymentLogo} />}
+      <View>
+        <H6>{props.header}</H6>
+        <Body>{props.body}</Body>
```

Comment:
The typographic style `Body` is incorrect because it's too big according to the design guidelines. You need to replace it with a `LabelSmall` one:
```suggestion
        <LabelSmall weight="Regular" color={theme["textBody-tertiary"]}>
            {props.body}
          </LabelSmall>
```